idd/Energy+.idd.in

```diff
@@ -24527,10 +24527,58 @@ ZoneEarthtube,
       \note  "C" in Equation
       \type real
       \default 0
- N18; \field Velocity Squared Term Flow Coefficient
+ N18, \field Velocity Squared Term Flow Coefficient
       \note  "D" in Equation
       \type real
       \default 0
+ A5,  \field Earth Tube Model Type
+      \type choice
+      \key Basic
+      \key Vertical
+      \default Basic
+ A6;  \field Earth Tube Model Parameters
+      \type object-list
+      \object-list EarthTubeParameterNames
+
+ZoneEarthtube:Parameters,
+      \memo Parameters that apply to the vertical model for an earth tube
+ \min-fields 6
+ A1,  \field Earth Tube Model Parameters Name
+      \required-field
+      \reference EarthTubeParameterNames
+ N1,  \field Nodes Above Earth Tube
+      \type integer
+      \units dimensionless
+      \minimum 3
+      \maximum 10
+      \default 5
+ N2,  \field Nodes Below Earth Tube
+      \type integer
+      \units dimensionless
+      \minimum 3
+      \maximum 10
+      \default 3
+ N3,  \field Earth Tube Dimensionless Boundary Above
+      \note When set to 1.0, the upper boundary is one earth tube radius below ground.
+      \type real
+      \units dimensionless
+      \minimum 0.25
+      \maximum 1.0
+      \default 1.0 
+ N4,  \field Earth Tube Dimensionless Boundary Above
+      \note When set to 1.0, the upper boundary is one earth tube radius below ground.
+      \type real
+      \units dimensionless
+      \minimum 0.25
+      \maximum 1.0
+      \default 0.25
```

Comment:
N4 is below. Note should be updated too.

Reply:
Yes, you are correct--N4 should say below.  Sorry!

@Myoldmopar Any chance we can slide in a typo correction to the field name in the IDD or is that not allowed.  Sorry about the bad cut/paste/non-edit.